resources/js/admin/settings/components/SettingText.vue

```diff
@@ -103,6 +103,10 @@ export default {
       this.$refs.input.focus();
     },
     onSave() {
+      if (this.setting.ui?.isNotEmpty && this.transformed == '') {
```

Comment:
Consider to use strict equality
```suggestion
      if (this.setting.ui?.isNotEmpty && (this.transformed === '' || this.transformed === null)) {
```